lib/components/RunFrameForCli/LeftHeader.tsx

```diff
@@ -85,15 +90,22 @@ export const RunframeCliLeftHeader = (props: {
       return
     }
     if (event.event_type === "REQUEST_EXPORT") {
-      setisExporting(true)
+      setIsExporting(true)
       setNotificationMessage("Export processing...")
       setIsError(false)
+      return
     }
-    if (event.event_type === "EXPORT_CREATED") {
-      setNotificationMessage(`Export created: ${event.exportFilePath}`)
-      setIsError(false)
-      setisExporting(false)
-    }
+  })
+
+  useExportHandler({
+    isExporting,
+    setIsExporting,
+    requestToExportSentAt,
```

Comment:
Hmm i think conventionally, isExporting and something like startExport would be returned